pixl_core/tests/test_project_config.py

```diff
@@ -67,6 +71,6 @@ def test_invalid_destinations():
 def test_invalid_paths():
     """Test that the config validation fails for invalid tag-operation paths."""
     config_data_wrong_base = BASE_YAML_DATA
```

Comment:
I expect you want to make a copy of this, or have the yaml data as a fixture otherwise if we tried to reuse it, the order of testing is important (would change items in the original `BASE_YAML_DATA` dict)?

Reply:
Fixed in #356